### DISCUSSION TITLE
Suggestion: add rectagular selection (to

### DISCUSSION THREAD

**Discussion by giwul70** [sumatrapdfreader/sumatrapdf#4650] [General]
I know it really is just a minor thing,but a positively meant suggestion. 
I was drafting a feature request post just now, suggestion to make it possible to rectangular select text, similar to the alt-left mouse combination in Acrobat. Created a screenshot of how it looks like in Acrobat, blah blah.
Checked out 'selection panel'(menu), no block/rectagular selection there. 
Then I just tried the ctrl-left mouse button and that worked...
Maybe add this to the menu?
(As said, it is just a minor thing)

![SnagIt-21112024 091503](https://github.com/user-attachments/assets/b27721b7-ad57-4cfb-976d-cb707aa037da)

Thumbs up for SumatraPDF!

**Comment by GitHubRulesOK** [comment]
Hmm the "selection" menu is what to DO with a selection set (not make one)
so there could be highlight, zoom or print/save included there but each have their own menus

Marquee in SumatraPDF is not similar to the one in Acrobat
![image](https://github.com/user-attachments/assets/28a21911-7428-4dc3-91cb-429143d09cd5)

**Comment by giwul70** [comment]
Thanks again.
As I was not aware of the rectangular selection within SumatraPDF I  used Acrobat to do the selection there and paste it into Excel (for example).
Anyway, matter solved.
